fdbserver/RestoreMaster.actor.cpp

```diff
@@ -174,10 +202,15 @@ ACTOR Future<Void> startProcessRestoreRequests(Reference<RestoreMasterData> self
 	// Step: Notify all restore requests have been handled by cleaning up the restore keys
 	wait(notifyRestoreCompleted(self, cx));
 
-	try {
-		wait(unlockDatabase(cx, randomUID));
-	} catch (Error& e) {
-		TraceEvent(SevWarn, "UnlockDBFailed").detail("UID", randomUID.toString());
+	numTries = 0;
+	loop {
+		try {
+			wait(unlockDatabase(cx, randomUID));
```

Comment:
To record off-thread conversation, remove the loop, and add an ASSERT_WE_THINK(false) in the catch.